pkg/executables/kubectl.go

```diff
@@ -780,6 +781,10 @@ func WithArgs(args []string) KubectlOpt {
 	return appendOpt(args...)
 }
 
+func WithArg(arg string) KubectlOpt {
```

Comment:
Why don't we define a new option/s for the arguments we need?

Reply:
This provides ability to use kubectlOpt with something like: get package my-flux. So in this case my-flux would be the arg. I think this follows the approach we have been trying to follow for KubectlOpt.